app/views/admin/AdministerUserModal.js

```diff
@@ -234,6 +235,42 @@ module.exports = (AdministerUserModal = (function () {
       return this.user.patch(options)
     }
 
+    onClickAddCreditsButton () {
+      const attrs = forms.formToObject(this.$('#credit-form'))
+      attrs.addCredits = parseInt(attrs.addCredits)
+      if (!_.all(_.values(attrs))) { return }
```

Comment:
what does this check do?

Reply:
It's used to check if any of the attributes are undefined but I don't think it's necessary now that I check them later in the code